core/src/main/javascript/crypto/EccCryptoContext.js

```diff
@@ -21,10 +21,22 @@
  * @implements {ICryptoContext}
  */
 var EccCryptoContext;
+var EccCryptoContext$Mode;
 
 (function() {
     "use strict";
 
+    /**
+     * ECC crypto context mode.
+     * @enum
+     */
+    EccCryptoContext$Mode = {
+        ENCRYPT_DECRYPT: 1,
+        SIGN_VERIFY: 2
+    };
+
+    var Mode = EccCryptoContext$Mode;
+
     EccCryptoContext = ICryptoContext.extend({
         /**
          * <p>Create a new ECC crypto context for sign using the provided private key.
```

Comment:
Update the documentation to match what the functionality is. See EccCryptoContext.java for an example.